install/bambuddy-install.sh

```
@@ -0,0 +1,69 @@
+msg_info "Installing Dependencies"
+$STD apt-get install -y \
+  libglib2.0-0
+msg_ok "Installed Dependencies"
+
+PYTHON_VERSION="3.13" setup_uv
+NODE_VERSION="22" setup_nodejs
+
+fetch_and_deploy_gh_release "bambuddy" "maziggy/bambuddy" "tarball" "latest" "/opt/bambuddy"
+
+msg_info "Setting up Python Environment"
+cd /opt/bambuddy
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)